### ISSUE TITLE
Village Flower, Specialty Flower, and Sister Fruit

### ISSUE THREAD

**Issue by inferno771** [kwsch/NHSE#202]
> notes -- 
fruit, flowers:
https://github.com/Treeki/CylindricalEarth/blob/d120acb8749860f52807325b896f0ab56ef90810/savefile/save_schema_131080_131078_pseudoC.h#L1520-L1527
birthday
https://github.com/Treeki/CylindricalEarth/blob/d120acb8749860f52807325b896f0ab56ef90810/savefile/save_schema_131080_131078_pseudoC.h#L306-L314

_Originally posted by @kwsch in https://github.com/kwsch/NHSE/issues/164#issuecomment-624375901_

Were these ever added to the editor? I'm only seeing the fruit setting under the 'Edit Misc' section.

**Comment by MirayXS** [comment]
Yes!

**Comment by MirayXS** [comment]
Map > Edit Map... > Field Goods

**Comment by inferno771** [comment]
> Map > Edit Map... > Field Goods

Is there a list of the flower ids somewhere for this?

**Comment by kwsch** [comment]
The Flower IDs for `SaveFg` have not been documented.

**Comment by MirayXS** [comment]
yeah. You can see them out in a spreadsheet in the other tab

**Comment by MirayXS** [comment]
Spreadsheet
https://docs.google.com/spreadsheets/d/13d_LAJPlxMa_DubPTuirkIV4DERBMXbrWQsmSh8ReK4/edit#gid=1275423323

**Comment by Soorily** [comment]
Sorry to piggyback onto this thread. I noticed it was asking for information that I am also looking for. I tried modifying the 'SpecialtyFruit' ID but my normal native fruit was still being sold in shop. I also tried changing the numbers for the flower sections but the seeds that are in my shop also remain the same. I couldn't find the information about where to locate the correct flower numbers in that document either. I did find the fruit internal IDs. Were you able to successfully change your shop fruit and flower seeds using the field goods section?
